src/user-guide/writing-workflows/external-triggers.rst

```diff
@@ -402,30 +402,12 @@ An example xrandom trigger workflow:
 Validation example using xrandom
 """"""""""""""""""""""""""""""""
 
-The ``xrandom`` xtrigger module contains a ``validate`` function.
+The ``xrandom`` xtrigger module contains an example of an xtrigger validation
```

Comment:
Can use `:py:mod:` to link to the source code?

Reply:
Already linked using the autodoc?